stdlib/3/multiprocessing/context.pyi

```diff
@@ -0,0 +1,98 @@
+# Stubs for multiprocessing.context
+
+from logging import Logger
+from multiprocessing import Process
+from typing import Any, Callable, Optional, List, Sequence, Tuple, Union
+
+class ProcessError(Exception): ...
+
+class BufferTooShort(ProcessError): ...
+
+class TimeoutError(ProcessError): ...
+
+class AuthenticationError(ProcessError): ...
+
+class BaseContext(object):
+    ProcessError = ...  # type: Exception
```

Comment:
`Type[Exception]` actually; Exception would be an exception instance, not an exception class. I'm not sure whether `ProcessError = ProcessError` would work.